internal/cli/pipeline_inspect.go

```diff
@@ -192,6 +192,10 @@ func (c *PipelineInspectCommand) Run(args []string) int {
 			},
 		}...)
 	} else {
+		if len(runs.PipelineRuns) == 0 {
+			c.ui.Output("No pipelines have ran", terminal.WithErrorStyle())
+			return 1
```

Comment:
We shouldn't err in this case, because we still want to be able to inspect a pipeline if it hasn't been run before, we just can't include pipeline run info. Maybe instead of `else` on 194 we do `else if len(runs.PipelineRuns) == 0`?

Reply:
That sounds reasonable to me